src/components/plots/piePlot.ts

```diff
@@ -24,6 +27,8 @@ export module Plots {
     constructor() {
       super();
       this._colorScale = new Scales.Color();
+      this._innerRadiusAccessor = () => 0;
+      this._outerRadiusAccessor = () => Math.min(this.width(), this.height()) / 2;
```

Comment:
We should use the actual `propertyAccessor()` calls here, to avoid inconsistencies.